.classpath

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not really a finding for this file but didn't know where to put it :).
As a rule of thumb (for which there can be exceptions, ofc) binary files should not be submitted. In order for git to ignore them and not consider them "submittable" changes, you can add a .gitignore file with the extensions of the files to be ignored (e.g. ".class") and even with whole folders to be ignored (e.g. out/).